py/torch_tensorrt/fx/utils.py

```diff
@@ -15,6 +16,45 @@
 from .types import Shape, TRTDataType
 
 
+class Frameworks(Enum):
+    NUMPY = "numpy"
+    TORCH = "torch"
+    TRT = "trt"
+
+
+DataTypeEquivalence: Dict[
```

Comment:
Unified data type translator dictionary which takes a `TRTDataType` and translates it to Numpy, Torch, or TRT data types of the equivalent type and precision.